app/views/user_mailer/email_confirmation_instructions.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="lead">
+  <%= t('user_mailer.email_confirmation_instructions.first_sentence.forgot_password_html') %>
+</div>
+
+<p>
+  <%= link_to t('user_mailer.email_confirmation_instructions.link_text'),
+              new_user_password_url,
+              target: '_blank',
+              style: "font-weight: bold;
+              display: inline-block;
+              padding: 0.75rem 1.5rem;
+              color: #ffffff;
+              background-color: #0071bb;
+              border-radius: 8px;
+              margin-bottom: 2em" %>
```

Comment:
this is to make the link a button, right? Could we use the same `<table class="button expanded large radius">` markup from the previous template?

If not, can we indent the continuing lines on this big string? I was very confused by it, I thought that `color: #fffff` was not going to work because it was missing a comma and isn't valid Ruby syntax

Reply:
I should have stuck with the table.  While it doesn't match the AC attachment and it's not semantic markup, it is consistent with other email messages.  I'll switch it back.